server/test/unit/server/managers/repo/test_publish.py

```diff
@@ -482,7 +482,7 @@ def test_publish_history_ascending_sort(self):
         date_string = '2013-06-01T12:00:0%sZ'
         for i in range(0, 10, 2):
             r = RepoPublishResult.expected_result(
-                'test_sort', 'test_dist', 'bar', date_string % str(i), date_string % str(i + 1),
+                'test_sort', 'test_dist', 'bar', {}, date_string % str(i), date_string % str(i + 1),
```

Comment:
This and many other tests have been updated to pass an empty dictionary here. Please make a new test that passes some data to this method and asserts that expected_result() did the right thing with the information.